src/core/version.ts

```diff
@@ -1,2 +1,2 @@
-let version = "1.0.0-b5";
+let version = "1.0.0-b6";
```

Comment:
Very minor; Can you read this from the package rather than repeating it?

Reply:
I'll get back to this. Don't want to introduce side effects by importing package.json without testing this out thoroughly.